src/Comments/Comments.js

```diff
@@ -45,6 +45,17 @@ export class Comments extends Component  {
       this.props.likeMovieComment(movieId, id, likeStatus)
   }
 
+    injectComments = () => {
+      return this.props.movieComments.map((comment, i) => {
+        return <Comment 
+          key={i} 
+          comment={comment}
+          likeComment={this.likeComment}
+          name={this.props.name}
+        />
+      })
+    }
+
```

Comment:
Nice! This seemed to get done pretty easily. I know we had 2 more places (MovieHeader and Card I think) that need the logic removed from their return statement as well.